packages/extension-api/src/extension-api.d.ts

```diff
@@ -101,6 +101,31 @@ declare module '@podman-desktop/api' {
     (listener: (e: T) => any, thisArgs?: any, disposables?: Disposable[]): Disposable;
   }
 
+  /**
+   * A class to create and manage an {@link Event} for clients to subscribe to.
+   * The emitter can only send one kind of event.
+   *
+   * Use this class to send events inside extension or provide API to the other
+   * extensions.
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  export class Emitter<T = any> {
```

Comment:
it should be `EventEmitter<T>`

https://github.com/microsoft/vscode/blob/main/src/vscode-dts/vscode.d.ts#L1606-L1624